### PR TITLE
Update 4_2_5_Processes.md

### PR DESCRIPTION
**Pull request by Sonia-IDSA**

### CODE REVIEW

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/4_2_5_Processes.md [hunk 1]
```diff
@@ -37,6 +40,16 @@ The certification process is divided into the following three phases:
     of the evaluation report by the certification body, which issues a
     certificate if the result of the evaluation process is positive.
 
+In the first assurance level certification, the applicant must apply for certification directly to the Certification Body  to trigger the start of the certification process. Once the Certification Body accepts the application, the applicant will conduct a self-assessment which will be evaluated by the Certification Body. If the result of the evaluation is possitive, the Certification Body will issue a certificate.
+
+![image](https://user-images.githubusercontent.com/77682996/155738445-479e9507-7415-4e16-b0dd-6d159d3d8528.png)
```
[the remark this answers is not in the document]
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/4_2_5_Processes.md]: thank you! I have slightly adjusted the images and added them directly to this pull requests

[the remark this answers is not in the document]
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/4_2_5_Processes.md]: thank you! I have slightly adjusted the images and added them directly to this pull requests

**Review by mohuber — APPROVED** [review]

**Comment by mohuber** [comment]
Hey @nadjamenz @ssteinbuss and @andreas-teuscher 
Sonia and I have finished the revision and review of this chapter but for merging the Pull Request we require a second approval. Could one of you maybe take a short look to the chapter and approve it? Thanks!

**Comment by nadjamenz** [comment]
@mohuber @Sonia-IDSA The changes look good, but there is still an open todo ([TODO: Add chapter on approval of evaluation facilities]). Will this be worked on after finalizing the current Pull Request?

**Comment by mohuber** [comment]
@nadjamenz thanks for the review!
The first process (Approval of evaluation facilities) has already been covered in another pull request: https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/pull/141.
Github should be able to automatically merge these together leaving no open points in the chapter.

**Review by nadjamenz — APPROVED** [review]
Changes approved, with the one open todo in this Merge Request being covered in #141.

**Comment by andreas-teuscher** [comment]
This changes are also fin for me.

**Comment by mohuber** [comment]
Thanks to both of you @nadjamenz and  @andreas-teuscher for the second review!

**Pull request merged**